pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -417,11 +417,13 @@ func (f *framework) RunScorePlugins(pc *PluginContext, pod *v1.Pod, nodes []*v1.
 	workqueue.ParallelizeUntil(ctx, 16, len(f.scorePlugins), func(index int) {
 		pl := f.scorePlugins[index]
 		nodeScoreList := pluginToNodeScores[pl.Name()]
-		status := pl.NormalizeScore(pc, pod, nodeScoreList)
-		if !status.IsSuccess() {
-			err := fmt.Errorf("normalize score plugin %q failed with error %v", pl.Name(), status.Message())
-			errCh.SendErrorWithCancel(err, cancel)
-			return
+		if extensions := pl.Extensions(); extensions != nil {
```

Comment:
Maybe you can combine this check with the one above in https://github.com/kubernetes/kubernetes/pull/83318/files#diff-76b7e314f9618c8bc2ca6442ddd35dc3R326? Might make it clearer that these are the same interface, especially since there it's just `extension` while this is`extensions`